packages/cli/core/src/commands/config/set.ts

```diff
@@ -1,7 +1,10 @@
 import {CliUx, Command, Flags} from '@oclif/core';
 import {Config} from '@coveo/cli-commons/config/config';
 import {AuthenticatedClient} from '@coveo/cli-commons/platform/authenticatedClient';
-import {IsAuthenticated, Preconditions} from '@coveo/cli-commons/preconditions';
+import {
+  IsAuthenticated,
+  Preconditions,
+} from '@coveo/cli-commons/preconditions/index';
```

Comment:
the subpath shenanigans in the packages ain't compatible with folder-as-module.
This is normal, considering that https://nodejs.org/api/modules.html#folders-as-modules is 'legacy'